app/views/orgs/_profile_form.html.erb

```diff
@@ -3,6 +3,7 @@
 <%
   shared_links_tooltip = _('Links will be displayed next to your organisation\'s logo')
   org_config_info_tooltip =  _('This information can only be changed by a system administrator. Contact the Help Desk if you have questions or to request changes.')
+  org_types_tooltip = _('This information can only be changed by a system administrator. Contact support@portagenetwork.ca if you have questions or to request changes.')
```

Comment:
Use `Rails.configuration.x.organisation.email` in this string with formatting like "some text %{value}" % { value: "text" }

Reply:
Ohh sorry i thought I already made this change...I'll fix it